src/INSTALL_NOTES

```diff
@@ -42,26 +42,19 @@ Unix (gzipped tar file):
 
 1.  Run the visit-install script to install visit.  If you have built your
     own visit binary distribution, you can find the visit-install script
-    in the svn_bin directory.  The visit-install script can create a new
-    distribution directory or update an existing distribution.  This is
-    handled automatically.
+    in the src/tools/dev/scripts/visit-install directory.  The visit-install
+    script can create a new distribution directory or update an existing
+    distribution.  This is handled automatically.
 
-        chmod 755 visit-install3_4_0
-        ./visit-install3_4_0 "version" "platform" "directory"
+        chmod 755 visit-install3_4_1
+        ./visit-install3_4_1 "version" "platform" "directory"
 
     where
 
-        "version" will be 3.4.0 for the current distribution.
+        "version" will be 3.4.1 for the current distribution.
 
-        "platform" will be one of the following:
+        "platform" will be the release relevant to your system, for example:
 
-            linux-x86_64-debian9
-            linux-x86_64-debian10
-            linux-x86_64-debian11
-            linux-x86_64-fedora31
-            linux-x86_64-rhel7
-            linux-x86_64-rhel7-wmesa
-            linux-x86_64-ubuntu18
             linux-x86_64-ubuntu20
```

Comment:
do we only want the one example here?

Reply:
I dont know what the full list will be until we arrive there